arXivDigest/api/app.py

```diff
@@ -2,16 +2,19 @@
 __author__ = "Øyvind Jekteberg and Kristian Gingstad"
 __copyright__ = "Copyright 2018, The ArXivDigest Project"
 
-import mysql.connector
-import sys
-import os
-from flask import Flask, g, jsonify, request, make_response
 from datetime import datetime
 
-import api.database as db
-from api.utils import validateApiKey, getUserlist
-import api.validator as validation
-from core.config import api_config
+from flask import Flask
+from flask import g
+from flask import jsonify
+from flask import make_response
+from flask import request
+
+import arXivDigest.api.database as db
```

Comment:
Python packages should have short, all-lowercase names. Rename arXivDigest to arxivdigest.
(It's only for naming the package. In the documentation, we still want to refer to it as arXivDigest)